joern-cli/frontends/csharpsrc2cpg/src/test/scala/io/joern/csharpsrc2cpg/passes/ast/DeclarationTests.scala

```diff
@@ -0,0 +1,47 @@
+package io.joern.csharpsrc2cpg.passes.ast
+
+import io.joern.csharpsrc2cpg.testfixtures.CSharpCode2CpgFixture
+import io.shiftleft.semanticcpg.language.*
+
+class DeclarationTests extends CSharpCode2CpgFixture {
+  "AST nodes for" should {
+    val cpg = code(basicBoilerplate("""
+          |int foo = 10; // When using basicBoilerplate, this is lineNumber 9 + (if any) number of global statements added
+          |string bar = "bar";
+          |foo + 5;
+          |""".stripMargin))
+
+    "identifiers should be created" in {
+      cpg.identifier.size shouldBe 3
```

Comment:
I recommend extracting the three identifies after this and testing them directly to avoid issues when other changes come in. E.g.
```
cpg.identifier.size shouldBe 3
val foo = cpg.identifier.nameExact("foo").head
foo.typeFullName shouldBe "System.Int32"
// etc.
```